model-optimizer/extensions/front/onnx/space_to_depth_ext.py

```diff
@@ -0,0 +1,31 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from extensions.ops.space_to_depth import SpaceToDepth
```

Comment:
What is the difference between different versions of this operation in ONNX? Should we support all of them?

Reply:
There are only opset-1 and opset-13 versions, and opset-13 is introduced only because it supports bfloat16 as input.